extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.Icons-vector.scss

```diff
@@ -391,6 +391,12 @@
 .oo-ui-icon-media {
 	@include ve-icon(media, wikia);
 }
+.oo-ui-icon-image {
+	@include ve-icon(image, wikia);
+}
+.oo-ui-icon-video {
```

Comment:
If it's not being used it would be good to just remove it - otherwise this break pretty nice relation of class name and file name, e.g. "oo-ui-icon-image" => "image"